src/index.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+// const temperature = 1  // for now CHANGE LATER
+// const state = {
+let tempIncreaseButton = null;
+let tempDecreaseButton = null;
+let getRealTimeTemp = null;
+let tempValue = null;
+let gardenContent = null;
+let temperature = 79;
+let headerCity = null;
+let cityNameUserInput = null;
+let defaultCityName = 'Princeton';
+let skyTemplate = null;
+let skyOption = null;
+let skyInput = null;
+let resetCityName = null;
+let realTimeTemperature = null;
+// }
+
+const selectTag = () => {
+    tempIncreaseButton = document.getElementById('increaseTempControl');
+    tempDecreaseButton = document.getElementById('decreaseTempControl');
+    tempValue = document.getElementById('tempValue');
+    gardenContent = document.getElementById('landscape');
+    headerCity = document.getElementById('headerCityName');
+    cityNameUserInput = document.getElementById('cityNameInput');
+    skyInput = document.getElementById('skySelect');
+    skyTemplate = document.getElementById('sky');
+    resetCityName = document.getElementById('cityNameReset');
+    realTimeTemperature = document.getElementById('currentTempButton');
+};
+//Create optionsa element for sky in the HTML file
+const createOptions = () => {
+    const optionArray = ['Sunny', 'Cloudy', 'Snowy', 'Rainy'];
+    for (let i = 0; i < optionArray.length; ++i) {
+        skyOption = document.createElement('option');
+        skyOption.textContent = optionArray[i];
+        skyInput.appendChild(skyOption);
+    }
+};
+
+const registerEvent = () => {
+    tempIncreaseButton.addEventListener('click', handleIncreaseTempClick);
+    tempDecreaseButton.addEventListener('click', handleDecreaseTempClick);
+    resetCityName.addEventListener('click', handleResetButton);
+    realTimeTemperature.addEventListener('click', handleRealTemp);
+    skyInput.addEventListener('change', skyInputHandler);
+    cityNameUserInput.addEventListener('input', headerCityFunction);
+};
+
+const handleIncreaseTempClick = () => {
+    ++temperature;
+    showTemp();
+};
+
+const handleDecreaseTempClick = () => {
+    --temperature;
+    showTemp();
+};
+
+
+
```

Comment:
Small nitpitck: we should review our code to ensure we're consistently applying spacing between functions across the project to avoid confusion from readers/maintainers around if functions are grouped for a reason or if something was added or removed and the spacing wasn't updated.